src/graphql/helpers.ts

```diff
@@ -83,6 +83,8 @@ export function formatSpace({
   space.voting.hideAbstain = space.voting.hideAbstain || false;
   space.voteValidation = space.voteValidation || { name: 'any', params: {} };
   space.delegationPortal = space.delegationPortal || null;
+  // disable boost by default until it's enabled
+  space.boost = space.boost || { disabled: true, bribeEnabled: false };
```

Comment:
Would it be more logical to have the field be `enabled` instead of `disabled`, so it's more consistent with `bribeEnabled` ?

Reply:
So i talked to @samuveth about it, https://github.com/snapshot-labs/snapshot.js/pull/975#discussion_r1524823252 The future plan is to enable it to everyone by default and only disable it to who want to disable it

Reply:
But if we have default values I can also call it `enabled`, and it could default to `true`

Reply:
> But if we have default values I can also call it `enabled`, and it could default to `true`

I prefer this